.php_cs

```diff
@@ -22,25 +22,31 @@ return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
-        'binary_operator_spaces' => ['align_double_arrow' => true, 'align_equals' => true],
-        'array_syntax' => array('syntax' => 'short'),
-        'blank_line_after_namespace' => true,
-        'header_comment' => array('header' => $header),
-        'lowercase_constants' => true,
-        'lowercase_keywords' => true,
-        'method_separation' => true,
-        'no_closing_tag' => true,
-        'no_extra_consecutive_blank_lines' => [
+        'array_syntax' => [
+            'syntax' => 'short'
+        ],
+        'binary_operator_spaces' => [
+            'align_double_arrow' => true,
+            'align_equals'       => true
+        ],
+        'class_attributes_separation' => [
```

Comment:
seems this this already the default value for @Symfony no ? 
https://github.com/FriendsOfPHP/PHP-CS-Fixer